src/collections/_documentation/development/server/environment.md

```diff
@@ -1,57 +1,38 @@
+To get started, clone the repo at [https://github.com/getsentry/sentry](https://github.com/getsentry/sentry) - or, your fork if you are not a member of `@getsentry`.
 
 ```bash
-git clone https://github.com/<your github username>/sentry.git
+git clone https://github.com/getsentry/sentry.git
 cd sentry
 ```
 
-{% capture __alert_content -%}
-It's important that you remain in the repo's root directory for the rest of the setup! Also, please do the following steps in the exact order they're shown.
-{%- endcapture -%}{%- include components/alert.html
-  title="Note"
-  content=__alert_content
-  level="warning"
-%}
-
+You're going to be working out of the repository for the remainder of the setup.
 
-### macOS - System Dependencies
+## System Dependencies
 
 Install [Homebrew](http://brew.sh), and then run `brew bundle` to install the various system packages as listed in sentry's `Brewfile`.
 
 One thing that requires manual attention is `docker`, which should have just been installed. Open up Spotlight, search for "Docker" and start it. You should soon see the docker icon in your macOS toolbar. Docker will automatically run on system restarts, so this should be the only time you do this.
 
-You can verify that docker is running by running `docker ps`. If it doesn't error with something like `Error response from daemon: dial unix docker.raw.sock: connect: connection refused`, you're good to continue.
-
+You can verify that Docker is running by running `docker ps`. If it doesn't error with something like `Error response from daemon: dial unix docker.raw.sock: connect: connection refused`, you're good to continue.
 
-### macOS - Python
+## Python
 
-Unfortunately, homebrew recently removed Python 2. While some versions of macOS ship with Python 2, it's recommended to not use the system's installation. Sentry also requires a specific version of Python, as shown in the file `.python-version`, and this may differ from the system's python.
+We utilize [pyenv](https://github.com/pyenv/pyenv) to install and manage python versions. It should have already been installed earlier when you ran `brew bundle`.
 
-We recommend using [pyenv](https://github.com/pyenv/pyenv) to install and manage python versions. It should have already been installed earlier when you ran `brew bundle`.
-
-You should be able to install python by running `pyenv install`. This will take a while, since your computer is actually compiling python!
+To install the required versions of Python you'll need to run `pyenv install`. This will take a while, since your computer is actually compiling python!
 
 After this, if you type `which python`, you should see something like `/usr/bin/python`... this means `python` will resolve to the system's python. You'll need to make some manual changes to your shell initialization files, if you want your shell to see pyenv's python.
```

Comment:
> If not, then I need to change my shell init files.

Correct. I'm trying to explain why, rather than just tell people to make those changes. Those changes follow right after that sentence.

Reply:
fwiw i want to kill all of this from the setup docs and just put a "Having trouble with pyenv? [GO TO PYENV FAQ PAGE]" so we can make this short and sweet